presto-main/src/main/java/io/prestosql/Session.java

```diff
@@ -789,11 +789,9 @@ public ResourceEstimateBuilder setPeakMemory(DataSize peakMemory)
         }
 
         /**
-         * Estimate the resource usage for an executed query.
-         * <p>
-         * Takes the estimate of the CPU runtime and memory used for an executed query.
+         * Constructs the object that contains the estimated resource usage of a query.
          *
-         * @return {@link ResourceEstimates} wich shows estimated resource usage for a query.
+         * @return an object that contains the estimated resource usage for a query.
```

Comment:
```suggestion
         * @return the estimated resource usage for a query
```
Please refrain from using periods at the end of `@return` tag descriptions.

Reply:
I forgot to remove the period, my mistake